recipes/eigen/bld.bat

```diff
@@ -0,0 +1,6 @@
+mkdir -p build
+cd build
+
+cmake -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% -DCMAKE_BUILD_TYPE=Release ..
+cmake --build . --target INSTALL --config Release
```

Comment:
Does this do the right thing for install? I copied this from somewhere else. So, I don't know. :confused: